node/src/actors/epoch_manager/mod.rs

```diff
@@ -126,6 +129,14 @@ impl EpochManager {
                 // Start checkpoint monitoring process
                 actor.checkpoint_monitor(ctx);
 
+                // Start ntp update process
+                {
+                    let mut global_ts = get_mut_global_timestamp();
+                    global_ts.server = config.ntp.server[0].clone();
```

Comment:
This will panic if there are no NTP servers in the config. Maybe in that case we should disable NTP?

Reply:
If there are not ntp servers in config file, default settings will be used

Reply:
I mean if someone explicitly sets

    servers = []

Reply:
Why would do that?